lib/workers/repository/init/config.ts

```diff
@@ -121,6 +123,11 @@ export async function mergeRenovateConfig(
     }
     logger.debug({ configFile, config: renovateJson }, 'Repository config');
   }
+  const cache = getCache();
+  cache.config = {
+    configFile,
+    contents: clone(renovateJson),
+  };
```

Comment:
Isn't a function like updateCacheConfig a better solution here?

Reply:
I'm still thinking about the best approach. If we use `updateX` then we have 4-5 and they are each just one-line wrappers?